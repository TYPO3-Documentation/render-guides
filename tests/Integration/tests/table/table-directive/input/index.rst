===============
Table directive
===============

.. contents::

Table with caption and column width
===================================

..  table:: Example Table
    :widths: 30, 70
    :width: 100%
    :caption: My table caption

    +--------------+--------------+
    | Data 1       | Data 2       |
    +==============+==============+
    | Row 1, Col 1 | Row 1, Col 2 |
    +--------------+--------------+
    | Row 2, Col 1 | Row 2, Col 2 |
    +--------------+--------------+
