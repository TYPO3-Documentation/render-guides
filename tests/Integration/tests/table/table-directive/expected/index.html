<!-- content start -->
                <section class="section" id="table-directive">
            <h1>Table directive<a class="headerlink" href="#table-directive" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h1>
            <div class="contents-wrapper">
        <ul class="menu-level">
    <li class="toc-item">
    <a href="#table-with-caption-and-column-width">Table with caption and column width</a>


</li>
    </ul>
</div>
            <section class="section" id="table-with-caption-and-column-width">
            <h2>Table with caption and column width<a class="headerlink" href="#table-with-caption-and-column-width" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>
            <div class="table-responsive">
    <table class="table table-bordered align-default colwidths-grid" style="width: 100%;">
            <caption>My table caption</caption>
            <colgroup>
                                    <col style="width: 30%">
                                                <col style="width: 70%">
                        </colgroup>
            <thead>
            <tr>
                            <th>Data 1</th>
                            <th>Data 2</th>
                    </tr>
        </thead>
        <tbody>
    <tr>
            <td>Row 1, Col 1</td>            <td>Row 1, Col 2</td>    </tr>    <tr>
            <td>Row 2, Col 1</td>            <td>Row 2, Col 2</td>    </tr></tbody>    </table>
</div>
    </section>
    </section>
        <!-- content end -->
